test/check-machines-lifecycle

```diff
@@ -89,6 +89,21 @@ class TestMachinesLifecycle(VirtualMachinesCase):
         self.waitPageInit()
         self.waitVmRow("subVmTest1")
 
+        # Check connection interpretation
+        b.click("#import-existing-vm")
+        b.wait_visible("#create-vm-dialog")
+        b.wait_visible('#create-vm-dialog button[aria-label="more info"]')
```

Comment:
Would it make sense to actually check what it does, i.e. that hovering over it creates a popup with some significant word or two in it?

Reply:
Make sense, I will complement the case, thanks